internal/pkg/service/common/etcdop/iterator/iterator.go

```diff
@@ -23,12 +23,13 @@ type Definition struct {
 }
 
 type Iterator struct {
-	config
+	config       config
 	ctx          context.Context
 	opts         []op.Option
 	client       etcd.KV
 	err          error
 	start        string         // page start prefix
+	end          string         // page start prefix
```

Comment:
Prepisoval sa `end` v config, tak som to radsej oddelil, je to cele dobre pokryte testami `iterator_test.go/iterator_typed_test.go`.